frontend/src/app/applications-upload/applications-upload.page.ts

```diff
@@ -0,0 +1,193 @@
+/**
+ * @description
+ * This page send the applications to the server
+ */
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { NavController } from '@ionic/angular';
+
+import { ToastController } from '../common/ion-wrappers/toast-controller';
+import {
+  EducationInfoEntry,
+  FormsData,
+  KeyCompetenciesEntry,
+} from '../model/forms-data.model';
+import { StorageHandlerService } from '../services/storage-handler.service';
+
+@Component({
+  selector: 'app-applications-upload',
+  templateUrl: './applications-upload.page.html',
+  styleUrls: ['./applications-upload.page.scss'],
+})
+export class ApplicationsUploadPage implements OnInit {
+  constructor(
+    private navController: NavController,
+    private storage: StorageHandlerService,
+    private httpClient: HttpClient,
+    private toastController: ToastController
+  ) {}
+  /**
+   * The total application size.
+   */
+  totalSize = 0;
+  /**
+   * The application size that is being uploaded.
+   */
+  uploadSize = 1;
```

Comment:
According to the text this size corresponds to uploading count. So shouldn't it be 0 initially?

Reply:
I feel we always start upload from 1 file and not 0 file so it is 1.